configs/LLNL-Sierra-IBM-power9-V100-Infiniband/auxiliary_software_files/packages.yaml

```diff
@@ -55,6 +55,11 @@ packages:
     - spec: cublas@11.8.0
       prefix: /usr/tce/packages/cuda/cuda-11.8.0
     buildable: false
+  fftw:
+    externals:
+    - spec: fftw@3.3.10
+      prefix: /usr/tcetmp/packages/fftw/fftw-3.3.10-xl-2023.06.28
+    buildable: false
   lapack:
```

Comment:
While this PR disables use of external blas/lapack, I think it would be worth reorganizing it:

```
  blas:
    buildable: false
  lapack:
    buildable: false
  netlib-lapack:
    externals:
    - spec: netlib-lapack@3.6.0
      prefix: /usr/tcetmp/packages/lapack/lapack-3.11.0-gcc-11.2.1/
```

(this config worked for me when using external blas/lapack and also avoids any potential issues with listing `lapack-xl` as an external, which isn't a Spack package).